src/signal.rs

```diff
@@ -55,6 +55,12 @@ use {
     WidgetHelpType,
 };
 
+#[cfg(gtk_3_16)]
+use {
+    gdk,
```

Comment:
This conditional import of `gdk` could be problematic. Since we're importing a lot of stuff from `gdk` above it's okay to import `GLContext` as well.